### PR TITLE
Adding instructions for manual installation

### PR DESCRIPTION
**Pull request by stephenhuang**

### CODE REVIEW

File: en-US/win10/SetupBoardManually.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep5.png">
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep6.png">    
+    </div>
+   </div>
+   
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">           
+      <li><b>Navigate to “C:\Program Files (x86)\Microsoft IoT\FFU and select Flash.ffu. </b></li>
```
Comment (bfjelds) [inline en-US/win10/SetupBoardManually.md]: C:\Program Files (x86)\Microsoft IoT\FFU<DeviceType>

File: en-US/win10/SetupBoardManually.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    <div class="col-md-6 col-sm-12 col-no-padding">   
+      <li><b>Launch the IoT Dashboard.</b> The Windows 10 IoT Core dashboard will allow you to flash the downloaded image onto your Sd card. You can download the IoT Dashboard <a href="http://go.microsoft.com/fwlink/?LinkID=708576" target="_blank">here.</a> </li>     
+    </div>
+    <div class="col-md-6 col-sm-12">
+    </div>
+   </div>    
+      
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">         
+      <li><b>Click ”Set up a new device”.</b> Once there, select “Custom” from the "Device type" drop down. Click the "Browse" button to find your .ffu </li>
```
Comment (bfjelds) [inline en-US/win10/SetupBoardManually.md]: Not sure if this is the correct way to say this or not (it absolutely could be), but the '.' in "to find your .ffu" seems odd to me.

**Comment by anthony-ngu** [comment]
Should these be addressed?

**Comment by stephenhuang** [comment]
Hm. Would could change it to be Click the "Browse" button to find your image (.ffu)
Does that make more sense?

**Comment by anthony-ngu** [comment]
@stephenhuang Seems good to me

**Comment by stephenhuang** [comment]
Addressed feedback. Ready to go up

**Comment by anthony-ngu** [comment]
@stephenhuang Sorry, just noticed this (the feedback probably hasn't gone out to the wider audience yet) if you could make sure that all file names and urls are lowercase with '-' separating words it would allow the future SEO transitions smoother.

File: en-US/win10/SetupBoardManually.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    </div>
+   </div>
+
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">
+      <li><b>Find the downloaded .ISO and double click on it.</b></li>
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep1.png">
```
Comment (anthony-ngu) [inline en-US/win10/SetupBoardManually.md]: Issue with capitalization

File: en-US/win10/SetupBoardManually.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    </div>
+   </div>
+   
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">   
+      <li><b>Double click the .MSI and follow the installation steps</b></li>
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep2.png">
```
Comment (anthony-ngu) [inline en-US/win10/SetupBoardManually.md]: Issue with capitalization

File: en-US/win10/SetupBoardManually.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep3.png">
```
Comment (anthony-ngu) [inline en-US/win10/SetupBoardManually.md]: Issue with capitalization

File: en-US/win10/SetupBoardManually.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    <div class="col-md-6 col-sm-12">
+    </div>
+   </div>    
+      
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">         
+      <li><b>Click ”Set up a new device”.</b> Once there, select “Custom” from the "Device type" drop down. Click the "Browse" button to find your image (.ffu) </li>
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep5.png">
```
Comment (anthony-ngu) [inline en-US/win10/SetupBoardManually.md:59]: Issue with capitalization

**Pull request closed without merging**